### ISSUE TITLE
Add VS2015 Support

### ISSUE THREAD

**Issue by tomspilman** [SickheadGames/Torsion#7]
Currently Torsion is setup to compile using Visual Studio 2003.

We need to replace the existing VS project and solution with one that works with the new VS2015 release.

We can consider adding support for older VS releases as well, but we need to keep it to a minimum.

**Comment by velteyn** [comment]
Hello, I put myself into this but I got a compiler error about this inport CMarkupSTL 
#include "MarkupSTL.h"
I cannot find
Is it a closed source library ?
i found it here Torsion\XmlFile.h

**Comment by tomspilman** [comment]
> Is it a closed source library ?

Yes... we have an open issue to replace it with TinyXML.  See https://github.com/SickheadGames/Torsion/issues/1.

**Comment by yurembo** [comment]
Hello all!
I have a Torsion licence.
Yep, some years ago I bought it from SickHead.
And now I wanna build Torsion from a source.
I installed VS 2003.
I included MarkupSTL.h and MarkupSTL.h to the project but after compillation I have an error: "Torsion fatal error LNK1104: cannot open file 'wxbase26d.lib'"
Do I need wxWidgets?

**Comment by yurembo** [comment]
I have compiled Torsion but this is shut down just after start

**Comment by velteyn** [comment]
Torsion was a commercial project before opening it up. Maybe some licensing code is still alive and wants a valid license.  Is your license correctly installed ? The license system , if present, should be removed !

**Comment by yurembo** [comment]
This question was asked a long long time ago...
However, yes, I have a correct Torsion license, and it works fine as before.
